Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -251,4 +321,41 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   }
 }
 
+RCT_EXPORT_METHOD(registerNotificationActionsForCategory:(NSDictionary*)actionsForCategory)
+{
+  if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 8.0 && actionsForCategory) {
+    UIMutableUserNotificationAction *action1;
+    action1 = [[UIMutableUserNotificationAction alloc] init];
+    [action1 setActivationMode:UIUserNotificationActivationModeBackground];
+    [action1 setTitle:actionsForCategory[@"firstActionTitle"]];
+    [action1 setIdentifier:actionsForCategory[@"firstActionId"]];
```

Comment:
could we do this as an array of dictionaries instead of a dictionary where the groupings are separated by `first` and `second` etc?